src/figures/figureS4.jl

```diff
@@ -201,14 +217,19 @@ lap100gem[1, :, 2], lap100gem[2, :, 2], _ = predict(LPT_GEM[:, 6, 7], LPT_GEM[:,
 lap100gem[3, :, :] .= lap100gem[1, :, :] .+ lap100gem[2, :, :]
 # well 2: 13, 19
 
-########### Lap 100 nM + palbociclibs [25 nM, 100 nM, 250 nM]
+########### Lap 100 nM + palbociclibs [25 nM, 50 nM, 100 nM, 250 nM]
 Lap100palbo = zeros(3, 189, 3)
 for i = 2:3
     Lap100palbo[1, :, i], Lap100palbo[2, :, i], _ = predict(LPT_PLB[:, 6, i + 4], LPT_PLB[:, 1, 1], t)
 end
 Lap100palbo[1, :, 1], Lap100palbo[2, :, 1], _ = predict(LPT_PLB[:, 6, 4], LPT_PLB[:, 1, 1], t)
 Lap100palbo[3, :, :] .= Lap100palbo[1, :, :] .+ Lap100palbo[2, :, :]
-# well 2: 8, 14, 20
```

Comment:
All the changes in `figureS4.jl` are to re-organize the data. Please ignore them.